src/modules/login/components/login_form.jsx

```diff
@@ -20,72 +24,25 @@ class LoginForm extends React.Component {
 		this.props.actions.login.bind(this)({
 			email: this.state.email,
 			password: this.state.password,
-			school: this.state.school || undefined,
```

Comment:
I removed it since we didn't use the parameter in the login action. However, we should actually pass it to the account service since there's a field `schoolId` in our model. I'll put it back :)